README.md

```diff
@@ -496,6 +496,12 @@ Language: English
 
 ### Java
 
+<img align="left" height="94px" width="94px" alt="Server Icon" src="images/server_icons/JavaDiscord.webp" />
+
+[__Java__](https://discord.gg/x65hqE6) [<img height="16px" width="16px" alt="Official Badge" src="images/badges/official.webp">](badges.md#official-identification-badge) [<img height="16px" width="16px" alt="Homepage URL" src="images/badges/homepage.webp">](https://javadiscord.net) [<img height="16px" width="16px" alt="Git Repository" src="images/badges/git.webp">](https://github.com/Java-Discord) \
+Notable Channels: `#help-guidelines`, `#rules`, `#expert-role`, `#self-roles` \
```

Comment:
Hello, thank you for contributing. Everything looks good except these 2 things:

- I haven't quite explained about "Notable Channels" on contribution guideline, it's been pretty confusing to many. I will update it soon enough. After reviewing your channels, I can see a few more that stand out as your community feature channels: `#question-of-the-week`, `#help`, `#code-review`, `#share-knowledge`, `#looking-for-programmer`, `#community-jam`. `#general` can be included too if it's a on-topic-only channel. Channels like, `#rules`, `#self-roles` are generic and common across all communities, your most active and productive servers are the notable channels🙂
- It seems you have added the official badge. I couldn't find your community in Oracle's official [community list](https://education.oracle.com/java-communities-resources). Do fix me if I'm wrong. Because you bear the word _Java_, which is a registered trademark of Oracle, you cannot be official using the same name. Your community resembles that of [Rust community](https://github.com/mhxion/awesome-discord-communities#rust), though very popular and very engaged with the official maintainers, not official yet. You can definitely PR anytime if the status changes. For now, it'd be more accurate if you removed the `[<img height="16px" width="16px" alt="Official Badge" src="images/badges/official.webp">](badges.md#official-identification-badge)` line.

Everything else about your community, its moderation, are great!